app/services/github/webhook_processors/create_project.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Github
+  module WebhookProcessors
+    class CreateProject
+      def initialize(body)
+        @wrapped_body = Github::Events::Installation.new(payload: body)
+      end
+
+      def call
+        user = find_user
+        project = find_project
+        ProjectUserRole.find_or_create_by(user: user, project: project).update!(role: ProjectUserRoleConstants::ADMIN)
+        @wrapped_body.repos.each { |repo_info| create_repo(repo_info, project) }
+        ReturnValue.ok(nil)
```

Comment:
Not a huge fun of returning `nil` as a successful result, have no idea what to use instead

Reply:
It should return at least `project`. It will be implemented with all the rest errors handling